### PR TITLE
page : quiz by Category added

### PR DESCRIPTION
**Pull request by manojsarna**
refactored code , some minor bug fixes

Added Quiz Category Page

Some Minor Bug Fixes

### CODE REVIEW

File: lib/Components/Category/tvshows/tvshows.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    .sm-main-cat-child {
+        width: 20rem;
+    }
+
+    .sm-main-cat-child img {
+        height: 20rem;
+    }
+}
```
Comment (ShrutiLakhani) [inline lib/Components/Category/tvshows/tvshows.css:68]: Clean code. Appreciate the way you have commented the entire code. Classes are named nicely

**Review by ShrutiLakhani — APPROVED** [review]
Everything is crisp and clean. Code looks perfect to merge.

I only feel that in dark mode, the category cards are merging with the background and aren't distinguished. But it's a personal choice.

File: lib/Components/Category/tvshows/tvshows.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                <div class="sm-quiz-card-container">
+                    <div class="sm-quiz-card-img">
+                        <img src="/lib/assets/images/tvshows/tv-4.webp" alt="tvshow">
+                        <div class="sm-quiz-card-title sm-grid-text">
+                            <span>Suits</span>
+                        </div>
+                    </div>
+                    <div class="sm-quiz-card-content">
+                        <h1 class="sm-quiz-card-subtitle">I dont play the odd's , I play the man.</h1>
+                        <h2 class="sm-quiz-card-info">Take this Quiz to test yourself!</h2>
```
Comment (pankaj-prog) [inline lib/Components/Category/tvshows/tvshows.html:61]: Do not use h2 for info, you can use p tag here

File: lib/Components/Category/tvshows/tvshows.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        padding: 0 1rem;
+    }
+
+    .sm-main-cat-child {
+        width: 20rem;
+    }
+
+    .sm-main-cat-child img {
+        height: 20rem;
+    }
```
Comment (pankaj-prog) [inline lib/Components/Category/tvshows/tvshows.css:67]: you can use a better name here, cat is not understood

File: lib/Components/Category/tvshows/tvshows.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
+@import "/lib/Components/core.css";
+@import "/lib/Components/utilities.css";
+@import "/lib/Components/Header/header.css";
+@import "/lib/Components/Footer/footer.css";
+@import "/lib/Components/QuizCard/quizcard.css";
+
+/* MAIN STARTS */
+.sm-main {
+    grid-area: main;
```
Comment (ritikaawasthy) [inline lib/Components/Category/tvshows/tvshows.css:9]: is grid area main needed here?

**Review by ritikaawasthy — APPROVED** [review]
Code looks clean and its easy to understand

File: lib/Components/Category/tvshows/tvshows.html [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+            <div class="sm-main-prod-container">
+                <div class="sm-quiz-card-container">
+                    <div class="sm-quiz-card-img">
+                        <img src="/lib/assets/images/tvshows/tv-4.webp" alt="tvshow">
+                        <div class="sm-quiz-card-title sm-grid-text">
+                            <span>Suits</span>
+                        </div>
+                    </div>
+                    <div class="sm-quiz-card-content">
+                        <h1 class="sm-quiz-card-subtitle">I dont play the odd's , I play the man.</h1>
```
Comment (pradipgharge) [inline lib/Components/Category/tvshows/tvshows.html:60]: just a typo, there shouldn't be space between comma and word.

**Review by pankaj-prog — APPROVED** [review]
UI looks amazing, Great work.

**Review by pradipgharge — APPROVED** [review]
The code looks nice, good work with the spacing. Looks good to merge!

**Pull request merged**